java/gazelle/private/servermanager/BUILD.bazel

```diff
@@ -12,3 +28,10 @@ go_library(
         "@rules_go//go/runfiles",
     ],
 )
+
+genrule(
```

Comment:
Should we use `load("@aspect_bazel_lib//lib:copy_file.bzl", "copy_file")` here?

Reply:
Good call! Done :)